build.gradle.kts

```diff
@@ -41,6 +41,7 @@ dependencies {
     implementation("org.slf4j", "slf4j-api")
     implementation("io.github.microutils", "kotlin-logging", prop("kotlinLoggingVersion"))
     compileOnly("org.slf4j", "slf4j-simple", prop("slf4jVersion"))
+    testRuntime("org.slf4j:slf4j-simple:${prop("slf4jVersion")}")
```

Comment:
what the heck?

Reply:
This makes it so drpc4k tests properly have the slf4j implementation on runtime path, otherwise it'd complain and use NO-OP.
I checked, and this does not leak into consumer's runtime at all (the resulting pom for the library is still the same).

You might not have had the issue before if you used IDEA without using separate modules per source sets (which makes it use a single module for both main and test, resulting in dependencies leaking from one to the other).

Reply:
but what about `compileOnly`? In my experience the main dependencies were also always available for testing?
Anyhow, please keep the notation consistent (commas, not colons).

Reply:
Fixed the notation, thought I did that but forgot about it.
`compileOnly` isn't enough when using separate modules per source sets (properly separating test from main dependencies)
Also, it was meant to replace `compileOnly` as we only need it to get logging when running the tests.